app/src/main/java/io/github/sspanak/tt9/ime/VoiceHandler.java

```diff
@@ -75,7 +75,14 @@ private void onVoiceInputStopped(String text) {
 
 
 	private void onVoiceInputError(VoiceInputError error) {
-		if (error.isIrrelevantToUser()) {
+		if (error.isLanguageMissing()) {
```

Comment:
On Android 12, just print the error message because automatic download is not possible